server/src/main/java/com/cloud/storage/secondary/SecondaryStorageVmManager.java

```diff
@@ -28,6 +30,12 @@
 
 public interface SecondaryStorageVmManager extends Manager {
 
+    static final ConfigKey<Long> SecondaryStorageCapacityScanInterval = new ConfigKey<>("Advanced", Long.class,
+            "secstorage.capacityscan.interval",
+            "30000",
+            "The time interval(in millisecond) to scan whether or not system needs more SSVM to ensure minimal standby capacity",
```

Comment:
should we make this seconds? miliseconds seems a bit overkill.

Reply:
can be changed but this is inline with the existing config for CPVM, 
```
    ConsoleProxyCapacityScanInterval(
            "Console Proxy",
            AgentManager.class,
            String.class,
            "consoleproxy.capacityscan.interval",
            "30000",
            "The time interval(in millisecond) to scan whether or not system needs more console proxy to ensure minimal standby capacity",
            null),
```